website/docs/docs/cloud/manage-access/set-up-sso-microsoft-entra-id.md

```diff
@@ -98,6 +98,10 @@ Under **Properties** check the toggle setting for **User assignment required?**
 | Microsoft Graph | Delegated | `Directory.Read.All` |
 | Microsoft Graph | Delegated | `User.Read` |
 
+:::info Why is `Directory.AccessAsUser.All` permission required?
+The reason why `Directory.Accessasuser.all` is required is because it lets dbt Cloud see what groups the user belongs to. dbt Cloud doesn't do anything more with those permissions. Setting up permissions this way lets the end users logging in not have to provide any more consent to get group info.
```

Comment:
```suggestion
`Directory.Accessasuser.all` is required is because it lets dbt Cloud see what groups the user belongs to. dbt Cloud doesn't use the permission for anything else. This setup avoids asking users for extra consent when they log in.
```